src/Functions.js

```diff
@@ -0,0 +1,18 @@
+export const dateToString = (date) => {
+  let newDate = new Date(date);
+  return newDate.toDateString().substring(4);
+};
+
+export const viewsToString = (views) => {
+  let newView = views / 1000;
+  return Math.round(newView) + "K";
+};
+
+export const titleShortner = (title) => {
+  let newTitle = title;
+  return newTitle.length < 58 ? newTitle : newTitle.substring(0, 55) + "...";
```

Comment:
avoid newTitle.length < 58 ? newTitle : newTitle.substring(0, 55) + "..."; make use of switch case

Reply:
I have used conditional operator. Since I am not using multiple cases for this use-case, I have implemented the conditional operator.